pkg/helpers/aws.go

```diff
@@ -359,7 +359,7 @@ func CloudWatchLogsStream(ctx context.Context, cw cloudwatchlogsiface.CloudWatch
 	var start int64
 
 	if opts.Since != nil {
-		start = time.Now().UTC().Add((*opts.Since)*-1).UnixNano() / int64(time.Millisecond)
+		start = time.Now().UTC().Add((*opts.Since)*-1).Unix() * int64(time.Microsecond)
```

Comment:
there is UnixMicro() bultin func in time pkg, use that?
```
// UnixMicro returns t as a Unix time, the number of microseconds elapsed since
// January 1, 1970 UTC. The result is undefined if the Unix time in
// microseconds cannot be represented by an int64 (a date before year -290307 or
// after year 294246). The result does not depend on the location associated
// with t.
func (t Time) UnixMicro() int64 {
	return t.unixSec()*1e6 + int64(t.nsec())/1e3
}
```

Reply:
sorry go 1.16 doesn't have UnixMicro() 
which version we are using?

Reply:
but i still think there is issue:
https://cs.opensource.google/go/go/+/refs/tags/go1.16.15:src/time/time.go;l=601
```
const (
	Nanosecond  Duration = 1
	Microsecond          = 1000 * Nanosecond
	Millisecond          = 1000 * Microsecond

```
calculation should be `Unix() * 1000000`

let's just use
```
.UnixNano() / int64(time.Microsecond)
```

Reply:
@nightfury1204 the problem was that, sometimes, with `time.Now().UTC()` we would get a high number, and dividing by `Microsecond` or `Millisecond` would cause it to round up to `1546300800001` instead of `1546300800000`.
I've tried to use both and it didn't work.